src/webgl/3d_primitives.js

```diff
@@ -1532,4 +1532,54 @@ p5.RendererGL.prototype.curveVertex = function() {
   }
 };
 
+p5.RendererGL.prototype.image = function(
+  img,
+  sx,
+  sy,
```

Comment:
I think this is a good start and a flat depth image rendering seems fine for now but might be worth considering if there should be an optional z-argument.

Reply:
If you don't feel like doing this now this could be a different issue/pull request.

Reply:
The z-argument will only define depth to the image right?

Reply:
Right so instead of passing 0 as the third argument every time that you call `this.vertex()` you would pass a z-value which determines depth.

Reply:
The signature of ```image()``` in WebGL mode will have an extra argument ```dz``` compared to 2D mode.
When ```image()``` is called by user, first the ```image()``` in loading_displaying.js is called which in turn calls the ```image()``` corresponding to the appropriate renderer. But now that the signature has changed, this will no longer be possible unless the argument ```dz``` is at end.

Reply:
That could be solved by checking whether the renderer is 3D and checking the argument length in `p5.prototype.image` in loading_displaying but again this could be a future addition to the WebGL image. It is fine to default to 0 depth for now.